### ISSUE TITLE
Tests!

### ISSUE THREAD

**Issue by lgrahl** [rawrtc/rawrtc-data-channel#15]
We need them!

**Comment by lxlenovostar** [comment]
If rawrtc-data-channel and rawrtc can support CMake, I can use it in the production environment faster.

**Comment by lgrahl** [comment]
I don't understand how that relates to unit and integration tests?

**Comment by lxlenovostar** [comment]
Can we use cross-compilation with meson?  I don't use meson before.

**Comment by lgrahl** [comment]
Yes, this is where Meson shines. See my various [CI images](https://github.com/rawrtc/ci-image) and the [Meson documentation](https://mesonbuild.com/Cross-compilation.html).

Please ask such questions in [our Gitter channel](https://gitter.im/rawrtc/Lobby) and not off-topic in this issue.
